_includes/themes/bootstrap/news/mica2-4.1-release.html

```diff
@@ -0,0 +1,14 @@
+<p xmlns="http://www.w3.org/1999/html">We are pleased to announce that Mica2 4.1 is now available. <a href="/pages/products/mica/">Mica</a> is OBiBa’s data web portal for epidemiological studies and networks.</p>
+
+<p>This release introduces the following new features:</p>
+
+<ul class="obiba-no-style-list">
+  <li><i class="fa fa-caret-right obiba-small-right-margin"></i>
+    <b>Study Import</b>, enables researchers to share study descriptions from remote Mica servers having identical data model - courtesy of <a href="https://recap-preterm.eu/" target="_blank">RECAP Preterm and EUCAN-Connect <i class="fa fa-external-link-square"></i></a>.
```

Comment:
Make a separate link for RECAP and EUCAN connect please

Reply:
Included in the 2nd commit